static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -21,18 +23,24 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   const {title, subtitle} = getTitleSubtitle(event);
   const avatarSize = parseInt(space(4), 10);
 
-  // If any of data fails to load, we don't want to render the component
-  // Only "One other issue appears in the same trace. View Full Trace (X issues)" would show up
+  const referrer = 'issue_details.related_trace_issue';
+
+  useRouteAnalyticsParams({has_related_trace_issue: true});
```

Comment:
Could you add `has_related_trace_issue: false` to the top-level call? That will ensure that amplitude records `false` instead of `(none)` when this component isn't rendered 

https://github.com/getsentry/sentry/blob/fb10e1f4cfabf57725ab1ae97ebf3d0a03034064/static/app/views/issueDetails/groupDetails.tsx#L589